src/app/app.component.ts

```diff
@@ -55,10 +55,10 @@ export class AppComponent implements OnInit, OnDestroy {
     this.checkLang();
   }
 
-  confirmed(event: any) {
-    event ?
+  optionTranslate(selected: boolean) {
+    selected ?
       ((this.selectedLanguage = this.language.id), this.setLanguage()) : console.log('Good for you');
```

Comment:
@Harshwardhan18 Please remove this console
This can just be a plain if statement